migrations/Version202312291225421874_ltiDeliveryProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\ltiDeliveryProvider\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\ltiDeliveryProvider\model\events\LtiAgsListener;
+use oat\oatbox\event\EventManager;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiTest\models\event\DeliveryExecutionFinish;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202312291225421874_ltiDeliveryProvider extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Remove TestVariablesRecorded and add DeliveryExecutionFinish event.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $eventManager = $this->getServiceLocator()->get(EventManager::SERVICE_ID);
+
+        $eventManager->attach(
+            DeliveryExecutionFinish::class,
+            [LtiAgsListener::class, 'onDeliveryExecutionFinish']
+        );
+
+        $eventManager->detach(
```

Comment:
did you check if this works?

Reply:
I've run the migration locally and `TestVariablesRecorded` is not present in `tao/config/generis/event.conf.php`

Reply:
I've run the migration locally and `TestVariablesRecorded` is not present in `tao/config/generis/event.conf.php`